lib/syscall_shim/uk_prsyscall.c

```diff
@@ -966,6 +1009,23 @@ static void pr_syscall(struct uk_streambuf *sb, int fmtf,
 
 #endif /* HAVE_uk_syscall_socket */
 
+#ifdef HAVE_uk_syscall_clone
+	case SYS_clone:
+		VPR_SYSCALL(sb, fmtf, syscall_num, args, PT_CLONEFLAGS,
+			    PT_VADDR, /* sp */
+			    PT_TID | PT_REF, /* ref to parent tid */
+#if CONFIG_ARCH_X86_64
```

Comment:
There is the following note from the static code analysis tool, Codacy:

```
failed to expand 'VPR_SYSCALL', it is invalid to use a preprocessor directive as macro parameter
```

Reply:
I see, interesting 🤔 . I can live with this. It seems it reduces the changes of mistakes if not done that way. I'll update it.